types/babel__standalone/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for @babel/standalone 7.1
+// Project: https://github.com/babel/babel/tree/master/packages/babel-standalone
+// Definitions by: Matheus Goncalves da Silva <https://github.com/PlayMa256>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9
+
+import { TransformOptions, types, FileResultCallback } from '@babel/core';
+
+export function transform(code: string, options: TransformOptions): string;
+
+export function transformFromAst(ast: types.Node, code: string | undefined, opts: TransformOptions | undefined, callback?: FileResultCallback): void;
+
+export function registerPlugin(name: string, plugin: object | (() => void)): void;
+
+export function registerPlugins(newPlugins: {
+    [key: string]: object | (() => void)
+}): void;
+
+export function registerPreset(name: string, preset: object | (() => void)): void;
+export function registerPresets(newPresets: {
+    [key: string]: object | (() => void),
+}): void;
+
+export const availablePlugins: Record<string, object | (() => void)>;
+export const availablePresets: Record<string, object | (() => void)>;
+
+export function transformScriptTags(scriptTags?: any[]): void;
```

Comment:
The source code ([`transformScriptTags`], [`runScripts`]) uses the `item` method which is not present in `Array`. I think there should be an `HTMLCollection`.

```suggestion
export function transformScriptTags(scriptTags?: HTMLCollection): void;
```

[`transformScriptTags`]: https://github.com/babel/babel/blob/master/packages/babel-standalone/src/index.js#L211
[`runScripts`]: https://github.com/babel/babel/blob/master/packages/babel-standalone/src/transformScriptTags.js#L185